ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -117,14 +117,6 @@
     <Reference Include="Ionic.Zip">
       <HintPath>..\packages\DotNetZip.1.9.3\lib\net20\Ionic.Zip.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Oops yes see you've upgraded. Unless critical to this specific PR would leave them as they were and submit a separate PR for the upgrade if it makes sense.